### PR TITLE
Fix fastp PF.

### PR DESCRIPTION
**Pull request by g-pacheco**
Screenshot (266)

- [ ] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` has been updated

### CODE REVIEW

**Review by ewels — APPROVED** [review]
Tested before and after the change and I can see no effect in the report, as expected. So I'm happy 👍🏻 

Please just add a line to the `CHANGELOG.md` file saying that you fixed a bug in the Fastp module with a link to the issue. Hopefully should be clear when you see how other changelog items are written.

Thanks!

Phil

**Comment by ewels** [comment]
Note (as this is your first ever PR) - no need to close the pull request or anything. Just commit + push the requested change to your branch and the pull request will automatically update with the additional changes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ### MultiQC updates
 
+- Fixed fastp `%PF` ([#1574](https://github.com/ewels/MultiQC/issues/1574))
```
Comment (ewels) [inline CHANGELOG.md:7]: Right idea but wrong place. Should be further down under `### Module updates` with a `- Fastp` bullet and then an indented bullet for the issue fixed..

Comment (g-pacheco) [reply CHANGELOG.md:7]: Sorry!!!

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,6 +29,8 @@
   - Fixed module failing when writing data to file if there is a `/` in the section name ([#1515](https://github.com/ewels/MultiQC/issues/1515))
 - **DRAGEN**
   - Fixed bug in sample name regular expression ([#1537](https://github.com/ewels/MultiQC/pull/1537))
+- **Fastp**
+    - Fixed Fastp `%PF` ([#1574](https://github.com/ewels/MultiQC/issues/1574))
```
Comment (ewels) [inline CHANGELOG.md:33]:
```suggestion
    - Fixed % pass filter statistics ([#1574](https://github.com/ewels/MultiQC/issues/1574))
```

File: CHANGELOG.md (hunk 1 above)
Comment (ewels) [reply CHANGELOG.md]: No problem!

**Comment by ewels** [comment]
1 commit for the actual bugfix, 5 commits for the changelog oneliner 😆 This is how it goes sometimes...

**Pull request merged**